### PR TITLE
wifi Configurations integrated to main branch, AWS sending issue fixed

### PR DESCRIPTION
**Pull request by divin-raj-kp**
we have fixed issue in sending data to AWS. Updated wifi configurations. @speethambaran @adhilxavier please review the changes.

### CODE REVIEW

File: nRF9160Master/src/PacketHandler/PacketHandler.c [hunk 1]
```diff
@@ -136,7 +162,11 @@ bool ProcessCmd(char *pcCmd)
         }
         else if(strstr(pcCmd, "ssid") != NULL)
         {
-            printk("Config: %s\n\r", pcCmd);
+            // printk("Config: %s\n\r", pcCmd);
```
Comment (adhilxavier) [inline nRF9160Master/src/PacketHandler/PacketHandler.c]: Avoid unwanted comments

File: nRF9160Master/src/WiFi/WiFiHandler.c [hunk 2]
```diff
@@ -13,21 +13,23 @@
 
 /*******************************************MACROS*********************************************************/
 #define MSG_SIZE 255
-#define WIFI_SSID_PWD       "realme GT 5G,s3qqyipp" //Change this line with SSID and password of choice
+//#define WIFI_SSID_PWD       "realme GT 5G,s3qqyipp" //Change this line with SSID and password of choice
```
Comment (adhilxavier) [inline nRF9160Master/src/WiFi/WiFiHandler.c]: Remove comment

File: nRF9160Master/src/WiFi/WiFiHandler.c [hunk 3]
```diff
@@ -13,21 +13,23 @@
...
 #define AWS_BROKER		    "a1kzdt4nun8bnh-ats.iot.ap-northeast-2.amazonaws.com"
 #define AWS_THING 		    "test_aws_iot"
 #define AWS_TOPIC 		    "test_aws_iot/testtopic"
 #define CFG_NUM 	        1
 #define CFG_NAME 	        "latlong"
 #define RETRY_COUNT         2
 
+char cWifiCredentials[80] = "Alcodex,Adx@2013";
```
Comment (adhilxavier) [inline nRF9160Master/src/WiFi/WiFiHandler.c]: Add a comment here to mention that it is SSID and password

File: nRf52840peripheral/src/PacketHandler/PacketHandler.c [hunk 4]
```diff
@@ -112,6 +112,7 @@ bool ProcessRcvdPacket(_sPacket *psPacket)
 */
 bool ProcessCmd(char *pcCmd)
 {
+    printk("\n 528ProcessCmd\n");
```
Comment (adhilxavier) [inline nRf52840peripheral/src/PacketHandler/PacketHandler.c]: Remove comment

File: nRf52840peripheral/src/PacketHandler/PacketHandler.c [hunk 5]
```diff
@@ -134,7 +138,7 @@ bool ProcessCmd(char *pcCmd)
 bool ProcessResponse(char *pcResp)
 {
     bool bRetVal = false;
-
+    printk("\n 528ProcessResp\n");
```
Comment (adhilxavier) [inline nRf52840peripheral/src/PacketHandler/PacketHandler.c]: Remove comment

File: nRf52840peripheral/prj.conf [hunk 6]
```diff
@@ -14,29 +14,26 @@ CONFIG_LOG=y
 CONFIG_PM=y
 CONFIG_PM_DEVICE=y
 #CONFIG_TRACING_SYSCALL=y
+
 #For enabling the definitionns of MTU exchange functions
 CONFIG_BT_GATT_CLIENT=y
-
+CONFIG_BT_BUF_EVT_RX_COUNT=5
```
Comment (speethambaran) [inline nRf52840peripheral/prj.conf:20]: what is the significance of this one ?

Comment (divin-raj-kp) [reply nRf52840peripheral/prj.conf:20]: this is for handling multiple incoming HCI events from the
Controller. This was in the main branch, I merged it with my branch Wifi Configuration.

File: nRf52840peripheral/src/PacketHandler/PacketHandler.c (hunk 5 above)
Comment (divin-raj-kp) [reply nRf52840peripheral/src/PacketHandler/PacketHandler.c]: will do that

File: nRf52840peripheral/src/PacketHandler/PacketHandler.c (hunk 4 above)
Comment (divin-raj-kp) [reply nRf52840peripheral/src/PacketHandler/PacketHandler.c]: will do that

File: nRF9160Master/src/WiFi/WiFiHandler.c (hunk 3 above)
Comment (divin-raj-kp) [reply nRF9160Master/src/WiFi/WiFiHandler.c]: will do that

File: nRF9160Master/src/WiFi/WiFiHandler.c (hunk 2 above)
Comment (divin-raj-kp) [reply nRF9160Master/src/WiFi/WiFiHandler.c]: will do that

File: nRF9160Master/src/PacketHandler/PacketHandler.c (hunk 1 above)
Comment (divin-raj-kp) [reply nRF9160Master/src/PacketHandler/PacketHandler.c]: will do that

File: .vscode/settings.json
Comment (adhilxavier) [inline .vscode/settings.json:1]: @divin-raj-kp Don't push settings.json file to repo. Get a new clone wifi configure branch delete the file and commit that change here.

Comment (adhilxavier) [reply .vscode/settings.json:1]: @divin-raj-kp Rest is fine you can merge this commit after doing the mentioned change.

Comment (divin-raj-kp) [reply .vscode/settings.json:1]: Ok

**Comment by divin-raj-kp** [comment]
please merge

**Comment by adhilxavier** [comment]
Merging this one and closing PR

**Pull request merged**